docs/proposals/20201020-management-cluster-operator.md

```diff
@@ -457,7 +457,7 @@ data:
 apiVersion: management.cluster.x-k8s.io/v1alpha1
 kind: InfrastructureProvider
 metadata:
- name: aws
+ name: myaws
```

Comment:
I will live aws here, I don't think we should encourage to use custom names even if in this case the fetch config is specified

Reply:
👍  I'll fix this one to aws.